spec/models/ncr/work_order_spec.rb

```diff
@@ -466,32 +468,37 @@
   describe "#current_approver" do
     it "returns the first pending approver" do
       wo = create(:ncr_work_order, :with_approvers)
-      expect(wo.current_approver).to eq(wo.individual_approvals.first.user)
+      expect(wo.current_approver).to eq(wo.approvers.first)
       wo.individual_approvals.first.approve!
-      expect(wo.current_approver).to eq(wo.individual_approvals.last.user)
+      expect(wo.current_approver).to eq(wo.approvers.second)
     end
 
     it "returns the first approver when fully approved" do
       wo = create(:ncr_work_order, :with_approvers)
-      wo.individual_approvals.first.approve!
-      wo.reload.individual_approvals.last.approve!
-      expect(wo.current_approver).to eq(wo.individual_approvals.first.user)
+      fully_approve(wo)
+      expect(wo.reload.current_approver).to eq(wo.approvers.first)
     end
   end
 
   describe "#final_approver" do
     it "returns the final approver" do
       wo = create(:ncr_work_order, :with_approvers)
-      expect(wo.final_approver).to eq(wo.individual_approvals.last.user)
+      expect(wo.final_approver).to eq(wo.approvers.last)
       wo.individual_approvals.first.approve!
-      expect(wo.final_approver).to eq(wo.individual_approvals.last.user)
+      expect(wo.final_approver).to eq(wo.approvers.last)
     end
- 
+
     it "returns the last approver when fully approved" do
       wo = create(:ncr_work_order, :with_approvers)
-      wo.individual_approvals.first.approve!
-      wo.reload.individual_approvals.last.approve!
-      expect(wo.final_approver).to eq(wo.individual_approvals.last.user)
-    end 
+      fully_approve(wo)
+      expect(wo.final_approver).to eq(wo.approvers.last)
+    end
+  end
+
+  def fully_approve(work_order)
```

Comment:
I had to invent this myself the other day for a different test. Perhaps this should be a method on WorkOrder itself?